test_sets/get_system_time.txt

```diff
@@ -0,0 +1,26 @@
+./test_scripts/Policies/GetSystemTime/001_GetSystemTime_is_sent.lua
```

Comment:
@GetmanetsIrina What about the script for the case #5 in first table on the page with checklist?